terraform/environments/tariff/locals.tf

```diff
@@ -38,5 +38,5 @@ locals {
   # environment specfic variables
   # example usage:  
   # example_data = local.application_data.accounts[local.environment].example_var
-  application_data = jsondecode(file("./application_variables.json"))
+  # application_data = jsondecode(file("./application_variables.json"))
```

Comment:
I don't mind the existing approach but I would prefer to use the terraform fileexists function to determine whether the application_variables.json exists before reading it: https://www.terraform.io/docs/language/functions/fileexists.html This way we would not need to programmatically comment it out in the provision-environment-directories.sh script. Other that that, the code changes seem fine.